tests/testthat/test-extract_tag.R

```diff
@@ -0,0 +1,31 @@
+context("extract tags")
+rd <- tempfile()
+
+test_that("multiline title tags are handled reasonably", {
+
+  writeLines("\\title{
+This is a nice title starting with a newline
+}", rd)
+
+  rd_file(rd) %>%
+    extract_tag("tag_title") %>%
+    expect_equal("This is a nice title starting with a newline")
+
+  writeLines("\\title{This is a title
+containing a newline}", rd)
+  
+  rd_file(rd) %>%
+    extract_tag("tag_title") %>%
+    expect_equal("This is a title\ncontaining a newline")
+})
+
```

Comment:
I think it would also be useful to include a test of a title with inline Rd, e.g. `\title{\emph{emph} and \strong{strong}}`